first-task/src/components/ModalAlbum/AddForm.jsx

```diff
@@ -0,0 +1,44 @@
+import React, {useContext, useState} from "react";
+import {Form, Button} from "react-bootstrap"
+import { AlbumContext } from "./AlbumContext";
+
+const AddForm = () => {
+    const {addAlbum} = useContext(AlbumContext);
+    const [addAlbumItem, setAddAlbumItem] = useState({
+        title:"", describe: ""
+    });
+
+    const addInput = (e) => {
+        setAddAlbumItem({...addAlbumItem,[e.target.name]: e.target.value})
+    }
+
+    const {title, describe} = addAlbumItem;
+
+    const handleSubmit = (e) => {
+        e.preventDefault();
+        addAlbum(title, describe);
+    }
+    return(
+        <Form onSubmit={handleSubmit}>
+            <Form.Group>
+                <Form.Control 
+                    placeholder="name"
+                    name="title"    
+                    value={title}
+                    required
+                    onChange = { (e) => addInput(e)}
```

Comment:
Again. Don't add spaces before and after `=`. And you can write just like `onChange={addInput}`